osbs/conf.py

```diff
@@ -227,6 +227,10 @@ def get_kerberos_principal(self):
     def get_kerberos_ccache(self):
         return self._get_value("kerberos_ccache", self.conf_section, "kerberos_ccache")
 
+    def get_cleanup(self):
+        return self._get_value("cleanup", self.conf_section, "cleanup",
```

Comment:
I'd suggest to use a less vague name and/or document the option in https://osbs.readthedocs.io/en/master/admins.html#configuring-osbs-client

Reply:
updated name and created docs PR